### ISSUE TITLE
PPM neuron conversion

### ISSUE THREAD

**Issue by Udit1999** [aws/aws-neuron-sdk#373]
Hey, previously I was able to compile the PPM module but now there is some bug with the interpolation
Here is my code sample

```
import torch.nn.functional as F
from torch import nn
import torch
import torch_neuron


class PPM(nn.Module):
    def __init__(self, in_dim, reduction_dim, bins):
        super(PPM, self).__init__()
        self.features = []
        for bin in bins:
            self.features.append(nn.Sequential(
                nn.AdaptiveAvgPool2d(bin),
                nn.Conv2d(in_dim, reduction_dim, kernel_size=1, bias=False),
                nn.BatchNorm2d(reduction_dim),
                nn.ReLU(inplace=True)
            ))
        self.features = nn.ModuleList(self.features)

    def forward(self, x):
        x_size = x.size()
        out = [x]
        for f in self.features:
            out.append(F.interpolate(f(x), x_size[2], mode='bilinear', align_corners=False))
        return torch.cat(out, 1)
       

model = PPM(256,128,[16,32,64])
model.eval() 

dummy_input = torch.randn(1,256, 128, 128)
model_neuron = torch_neuron.trace(model, dummy_input)

```

Is there anything I'm doing wrong?

**Comment by aws-taylor** [comment]
Hello @Udit1999,

 I have managed to reproduce the issue and am investigating. We will update this issue as more information becomes available.

-Taylor

**Comment by aws-taylor** [comment]
Hello @Udit1999,

 We've narrowed down the issue to the `nn.AdaptiveAvgPool2d(bin)` line. We're still investigating _why_ that line is an issue, but in the mean time removing it mitigates the issues and may allow you to make forward progress.

-Taylor

**Comment by bluesky314** [comment]
Hi @aws-taylor, any progress on this issue?

**Comment by awsrjh** [comment]
HI bluesky314

We believe we have a solution to this and expect in one of our upcoming releases.

**Comment by aws-zejdaj** [comment]
Hi Udit, We have implemented the solution, we are testing it it internally, and expect it to be available in the next release.

**Comment by Udit1999** [comment]
Hi @aws-zejdaj ,
Thanks for the update.

**Comment by mrnikwaws** [comment]
Hi Udit,

This problem has now been addressed.  I've confirmed it works correctly now on pre-release builds.  You should see it fixed in an upcoming release at which point we'll comment on this issue.

**Comment by Udit1999** [comment]
Hi @mrnikwaws, Thanks for the update.

**Comment by micwade-aws** [comment]
@Udit1999 - this issue was confirmed as resolved with Neuron 1.18 release: 
https://awsdocs-neuron.readthedocs-hosted.com/en/latest/release-notes/index.html#neuron-1-18-0-03-25-2022 

Please reopen if you still see a problem.
